client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -293,6 +293,7 @@ export const useCommandsArrayWpcom = ( {
 					clearEdgeCache( site.ID );
 				},
 				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+				filterNotice: __( 'Only listing sites with hosting features enabled.' ),
```

Comment:
[Suggestion/Proposal]: I suggest using a mapping object between the  `calypso-products/src/constants/features.ts` and the wording here to make more clear of what is the feature that makes this wording appear and avoid magic strings. For example ( this can be restructured based on what seems a better fit):

```javascript
const FilterNoticeMessage = {
  [ features.FEATURE_HOSTING] = 'hosting'
}
```
This can apply to all `filterNotice` properties below

Reply:
[Suggestion/Proposal]: We can also change `filterNotice` to be a function that builds the notice message with potentially more than one of the features in the wording. eg:

```javascript
filterNotice: ( noticeMessages ) => __(`Only listing sites with ${noticeMessages.join(',')} features enabled.`)
```
And use it like so:
```javascript
filterNotice( FilterNoticeMessage[ features.FEATURE_HOSTING]], FilterNoticeMessage[ features.OTHER_FEATURE]]
```

Reply:
Thanks for the suggestion, @kozer !